go-chaos/internal/zeebe.go

```diff
@@ -130,25 +131,45 @@ func extractNodeId(topologyResponse *pb.TopologyResponse, partitionId int, role
 //go:embed bpmn/*
 var bpmnContent embed.FS
 
-func DeployModel(client zbc.Client) error {
-
-	processModelFileName := "bpmn/one_task.bpmn"
-	bpmnBytes, err := bpmnContent.ReadFile(processModelFileName)
+func DeployModel(client zbc.Client, fileName string) error {
+	bpmnBytes, err := readBPMNFileOrDefault(fileName)
 	if err != nil {
 		return err
 	}
 
-	response, err := client.NewDeployProcessCommand().AddResource(bpmnBytes, processModelFileName).Send(context.TODO())
+	response, err := client.NewDeployProcessCommand().AddResource(bpmnBytes, fileName).Send(context.TODO())
 	if err != nil {
 		return err
 	}
 
 	if Verbosity {
-		fmt.Printf("Deployed process model %s successful with key %d.\n", processModelFileName, response.Processes[0].ProcessDefinitionKey)
+		fmt.Printf("Deployed process model %s successful with key %d.\n", fileName, response.Processes[0].ProcessDefinitionKey)
 	}
 	return nil
 }
 
+// if file not exist we read our default BPMN process model and return the content
+func readBPMNFileOrDefault(fileName string) ([]byte, error) {
+	var bpmnBytes []byte
+	var err error
+
+	if len(fileName) == 0 {
+		fileName = "bpmn/one_task.bpmn"
```

Comment:
:wrench: I would add verbose logging that we're falling back to the default process since nothing was given. It might seem redundant to you here because we also log later which process we use, but if someone doesn't know the internals of zeebe-chaos then they wouldn't know why this was picked.